cartridge/clusterwide-config.lua

```diff
@@ -249,6 +250,37 @@ clusterwide_config_mt = {
             return self._checksum
         end,
 
+        new = function(self, patch)
```

Comment:
I suggest to rename it to "patch". But I like this change - we need it in TDG code.

Reply:
Is "patch_copy" ok? What do you think?

Reply:
Returning back to https://github.com/tarantool/cartridge/pull/1438#discussion_r647789718 I don't understand why "copy" word is needed. But up to you.